page-index/page-index.js

```diff
@@ -658,6 +658,11 @@ const pageIndex = [
             type: ANCHOR,
             href: "#docker-wale",
           },
+          {
+            Title: "Named restore point",
```

Comment:
We probably want to consistently use a common term for this.  Restore points?  Consistent restore points?  Named restore points?  something else?  Below we use multiple ones.

(Is one widely used already from DB / postgres literature here?)

Reply:
@pmwkaa Mike has a point we should be consistent.